source/construct-languages.rq

```diff
@@ -22,17 +22,20 @@ construct {
             ?s rdfs:label ?label .
             optional { 
                 values ?n { 0 1 2 3 4 5 }
+                bind(concat("^([^|]+\\|){", str(?n) ,"} *") as ?skipN)
                 ?s skos:altLabel ?altLabels
-                bind(concat("^([^|]+\\|){", str(?n) ,"}") AS ?skipN)
-                bind(replace(replace(?altLabels, ?skipN, ""), "\\|.*$", "") AS ?altLabel)
+                bind(replace(replace(?altLabels, ?skipN, ""), " *\\|.*$", "") as ?altLabel)
             }
             optional { ?s rdfs:comment ?comment }
         } union {
             ?s skos:notation ?not .
             bind(strdt(?not, dc:ISO639-2) as ?langCode)
             bind(iri(concat('http://id.loc.gov/vocabulary/iso639-2/', ?not)) as ?loclang)
             graph <http://id.loc.gov/vocabulary/languages> {
-                ?loclang skos:prefLabel ?prefLabel .
+                ?loclang skos:prefLabel ?prefLabels
+                values ?n { 0 1 2 3 4 5 }
+                bind(concat("^([^|]+\\|){", str(?n) ,"} *") as ?skipN)
+                bind(replace(replace(?prefLabels, ?skipN, ""), " *\\|.*$", "") as ?prefLabel)
```

Comment:
This works but is super slow (1m20s on my laptop)
Should be possible to make it conditional on `contains(?prefLabels, "|")`??